bips-ec-ops.md

```diff
@@ -0,0 +1,1268 @@
+```
+  BIP: ???
+  Layer: Consensus (soft fork)
+  Title: Elliptic Curve Operations for Bitcoin Script
+  Author: Olaoluwa Osuntokun <laolu32@gmail.com>
+  Comments-Summary: X
+  Comments-URI: X
+  Status: Draft
+  Type: Standards Track
+  Created: 2025-08-22
+  License: BSD-3-Clause
+  Requires: 340, 341, 342
+```
+
+
+# Introduction
+
+## Abstract
+
+This document specifies a series of Elliptic Curve opcodes for secp256k1. These
```

Comment:
Suggest listing the new opcodes here.

```suggestion
This document specifies a series of Elliptic Curve opcodes for secp256k1: `OP_EC_POINT_ADD`, `OP_EC_POINT_MUL`, `OP_EC_POINT_NEGATE`, and `OP_EC_POINT_X_COORD`. These
```

(nit, use consistenc spelling of "opcodes" or "op codes" in this document)